app/models/education_navigation_ab_test_request.rb

```diff
@@ -8,7 +8,7 @@ class EducationNavigationAbTestRequest
   def initialize(request, content_item)
     @content_item = content_item
     @ab_test = GovukAbTesting::AbTest.new("EducationNavigation", dimension: 41)
-    @requested_variant = @ab_test.requested_variant request
+    @requested_variant = @ab_test.requested_variant(request.headers)
```

Comment:
Apologies @mcgoooo , those docs are now outdated. I will fix that in a different PR.

The gem `govuk_ab_testing` only needs information in the request headers, so only passing that in makes it clearer what the API needs. Furthermore, it makes testing the A/B tests easier as different acceptance testing frameworks (capybara, etc) treat a request object differently, so accessing the headers is widely different).

Does that make sense?